src/Components/Blueprints/BlueprintDiffModal.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { DiffEditor } from '@monaco-editor/react';
+import { Modal, ModalVariant } from '@patternfly/react-core';
+
+import { selectSelectedBlueprintId } from '../../store/BlueprintSlice';
+import { useAppSelector } from '../../store/hooks';
+import { useGetBlueprintQuery } from '../../store/imageBuilderApi';
+
+type blueprintDiffProps = {
+  // baseVersion is the version of the blueprint to compare the latest version against
+  baseVersion: number | null | undefined;
+  isOpen: boolean;
+  onClose?: () => void;
+};
+
+const BlueprintDiffModal = ({
+  baseVersion,
+  isOpen,
+  onClose,
+}: blueprintDiffProps) => {
+  const selectedBlueprintId = useAppSelector(selectSelectedBlueprintId);
+
+  const { data: baseBlueprint } = useGetBlueprintQuery(
+    { id: selectedBlueprintId as string, version: baseVersion || -1 },
+    { skip: !selectedBlueprintId || !baseVersion }
+  );
+  const { data: blueprint } = useGetBlueprintQuery(
+    { id: selectedBlueprintId as string },
+    { skip: !selectedBlueprintId }
+  );
+
+  if (!baseBlueprint || !blueprint) {
+    return null;
+  }
+
+  return (
+    <Modal
+      variant={ModalVariant.large}
+      // titleIconVariant="warning"
```

Comment:
Can we remove this or is it a placeholder?

Reply:
oh yeah, lets remove! :)

Reply:
Cool, I'll be ready for re-review :+1: